pkg/controllers/cluster/taint_manager.go

```diff
@@ -268,8 +264,8 @@ func (tc *NoExecuteTaintManager) syncClusterBindingEviction(key util.QueueKey) e
 }
 
 // needEviction returns whether the binding should be evicted from target cluster right now.
-// If a toleration time is found, we return false along with a minimum toleration time as the
-// second return value.
+// If a toleration time is found, we return false expect the toleration time is 0 in which case
```

Comment:
`expect`? or `except`? That's quite different.

Reply:
Sorry, I made a mistake; it should be "except"